From 15fedf5f425b562b5793e430ef422b0bbf1050ed Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Wed, 18 May 2016 06:59:47 +0000
Subject: [PATCH] abuild: fix apk_up2date check

fix check that all subpackages exists, not only main package
---
 abuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/abuild.in b/abuild.in
index d6cdf6c..db82cfd 100644
--- a/abuild.in
+++ b/abuild.in
@@ -1622,7 +1622,7 @@ apk_up2date() {
 	local i s
 	cd "$startdir"
 	for i in $pkgname $subpackages; do
-		[ -f "$PKGDEST/$pkgname-$pkgver-r$pkgrel.apk" ] || return 1
+		[ -f "$PKGDEST/${i%:*}-$pkgver-r$pkgrel.apk" ] || return 1
 	done
 	[ -n "$keep" ] && return 0
 
-- 
2.8.2

